cli/lockfile.rs

```diff
@@ -577,10 +581,12 @@ mod tests {
       id: NpmPackageId {
         name: "picocolors".to_string(),
         version: NpmVersion::parse("1.0.0").unwrap(),
+        peer_dependencies: Vec::new(),
       },
+      copy_index: 0,
```

Comment:
I suppose this doesn't warrant bumping lockfile version?

Reply:
The copy_index is derived from the package ids when loading the lockfile. For the peer dependency information now being stored in the id text, I think it's ok to not bump given npm specifiers are unstable, but old versions of deno will error loading it I think.